packages/x-tree-view/src/RichTreeView/RichTreeView.tsx

```diff
@@ -232,6 +232,11 @@ RichTreeView.propTypes = {
    * @returns {boolean} `true` if the item should be disabled.
    */
   isItemDisabled: PropTypes.func,
+  /**
+   * Indentation in pixels between two an item and its children.
+   * @default 12
+   */
+  itemChildrenIndentationPx: PropTypes.number,
```

Comment:
I am not sure this should **always** be in px 🤔 
How would we support other units? Would we have a prop for each? I think having this prop called `itemChildrenIndentation` and allowing for different units as well (with a fallback to px) is a far better dx - but maybe I'm missing something in terms of feasibility

Reply:
There is one place in the drag & drop where we need to use the value in pixels:

```tsx
      if (validActions['move-to-parent'] && cursorX < params.itemChildrenIndentationPx) {
        action = 'move-to-parent';
      }
```

If we can convert the value it's fine, but it would be hard to support all units I guess.

Reply:
Not sure how we could handle this, because I do agree with you that forcing the usage of pixels it not optimal

Reply:
> If we can convert the value it's fine

Oh, I see. 🤔 There is a way using a temporary element and `getComputedStyle` but it's kind of hacky IMO 🙈 
We can go with the px value for now I think

Reply:
> it's kind of hacky

And for stuff like `%` or `em` it would still be problematic :grimacing:

Reply:
Yes, true 🙈